openlibrary/templates/home/index.html

```diff
@@ -3,6 +3,18 @@
 $ _x = ctx.setdefault('bodyid', 'home')
 $var title: $_("Welcome to Open Library")
 
+$ meta_description = "Open Library is an open, editable library catalog, building towards a web page for every book ever published."
+$ meta_image_url = "https://openlibrary.org/static/images/ol-banner.png"
```

Comment:
This still needs the ol-banner.png included

Reply:
OL banner added